### ISSUE TITLE
Common discussion

### ISSUE THREAD

**Issue by Mosaddeq107** [ubsuny/23-Homework1G5#7]
@LinxuanHu Hello, we need to start doing our own work. Before starting to do so, it's important to study the requirements that the professor set. Are facing any issues?

**Comment by LinxuanHu** [comment]
i have some trouble with the quantum computer because it is quite differnet from a traditional computer

**Comment by Mosaddeq107** [comment]
we can discuss...
Please let me know in another issue. I am closing this issue.
